db_dtypes/pandas_backports.py

```diff
@@ -80,10 +79,16 @@ def _cmp_method(self, other, op):  # pragma: NO COVER
         return NotImplemented
 
 
-# TODO: use public API once pandas 1.5 / 2.x is released.
+# TODO: use public API once NDArrayBackedExtensionArray is added to the
+# pandas.
```

Comment:
It is in pandas. That's what `import_default` is doing. It's importing from `pandas.core.arrays._mixins`. We don't need this class anymore. Just use `pandas.core.arrays._mixins.NDArrayBackedExtensionArray` directly.

https://github.com/pandas-dev/pandas/blob/7595ed503df845f3bdf4d26b9beea5ec031b619d/pandas/core/arrays/_mixins.py#L93

Reply:
Used the suggested class directly.